webapp/src/components/call_widget/participant.tsx

```diff
@@ -0,0 +1,161 @@
+                    padding: '0',
+                    lineHeight: '20px',
+                    fontSize: '14px',
+                }}
+            >
+                {getUserDisplayName(profile)}
+            </span>
+
+            {(isYou || isHost) &&
+                <span style={{marginLeft: -8, display: 'flex', alignItems: 'baseline', gap: 5}}>
+                    {isYou &&
+                        <span style={youStyle}>
+                            {formatMessage({defaultMessage: '(you)'})}
+                        </span>
+                    }
+                    {isHost &&
+                        <HostBadge
+                            data-testid={'participant-list-host-badge'}
+                            onWhiteBg={true}
+                        />
+                    }
+                </span>
+            }
+
+            <span
+                style={{
+                    display: 'flex',
+                    justifyContent: 'center',
+                    alignItems: 'center',
+                    marginLeft: 'auto',
+                    gap: '14px',
+                }}
+            >
+                {session?.reaction &&
+                    <Emoji
+                        emoji={session.reaction.emoji}
+                        size={14}
+                    />
+                }
+                {isHandRaised &&
+                    <HandEmoji
+                        fill='var(--away-indicator)'
+                        style={{width: '14px', height: '14px'}}
+                    />
+                }
+
+                {isSharingScreen &&
+                    <ScreenIcon
+                        fill={'rgb(var(--dnd-indicator-rgb))'}
+                        style={{width: '14px', height: '14px'}}
+                    />
+                }
+
+                {showHostControls &&
+                    <StyledDotMenu
+                        icon={<StyledThreeDotsButton/>}
+                        dotMenuButton={DotMenuButton}
+                        dropdownMenu={StyledDropdownMenu}
+                        title={formatMessage({defaultMessage: 'Host controls'})}
+                        placement={'top-start'}
+                        portal={false}
+                        strategy={'fixed'}
+                        offset={0}
+                    >
+                        <HostControlsMenu
+                            callID={callID}
+                            userID={session.user_id}
+                        />
+                    </StyledDotMenu>
+                }
+
+                <MuteIcon
+                    fill={isMuted ? 'rgba(var(--center-channel-color-rgb), 0.56)' : '#3DB887'}
+                    style={{width: '14px', height: '14px'}}
+                />
+
+            </span>
+        </li>
+    );
+};
+
+const StyledDotMenu = styled(DotMenu)`
+    margin-right: -4px;
+`;
+
+const StyledThreeDotsButton = styled(ThreeDotsButton)`
+    fill: rgba(var(--center-channel-color-rgb), 0.56);
+`;
```

Comment:
Here and in other files, do you mind if we start using [transient props](https://styled-components.com/docs/faqs#transient-props-since-5.1) for styled components so we don't get DOM warnings? Prefixing passed props with `$` should be enough, same as I did in https://github.com/mattermost/mattermost-plugin-calls/pull/679 for the rest of them.

Reply:
K, sounds good, done.